bin/linkfix.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ "$1" == "-h" -o "$1" == "--help" ]; then
+    echo "$0 [-h] REPO"
+    echo "Normalizes the symlinks in the raw subdirectory of the repo"
+    echo "by linking to the ultimate destination."
+    echo "With -h, uses hard links instead of symlinks, allowing the"
+    echo "originals to be removed if desired."
+    exit 0
+fi
+symlink="-s"
+if [ "$1" == "-h" ]; then
+    symlink=""
```

Comment:
This is never reached.  If you use "-h" on the command line, the help message is printed, and exits.

Reply:
Oops.  Used the same option name twice.  My bad.